src/document/__tests__/DocumentAnnotator-test.ts

```diff
@@ -1,4 +1,5 @@
 import BaseManager from '../../common/BaseManager';
+import { ANNOTATION_CLASSES } from '../../common/BaseAnnotator';
```

Comment:
🔤  this should move below with the other non-default exports in alphabetical order

Reply:
Thanks for pointing this out!